accounts/views/connection_views.py

```diff
@@ -55,8 +48,9 @@ def update_connection(request, connection_id):
 
 def retrieve_connections(request):
     account = request.user
-    connections = list(account.connections.all())
-    return JsonResponse({"data": connections}, status=200)
+    connections = account.connections.all()
```

Comment:
* This will retrieve all the connections of all users,  we have just to retrieve the all connection of the authenticated user
* account.connections.filter(account=account)